pages/1.11/deploying-services/fault-domain-awareness/index.md

```diff
@@ -59,9 +59,22 @@ You can deploy your instances in a specific region region based on the available
 
 1. Create a fault domain detect script to run on each node to detect the node's fault domain (Enterprise only). During installation, the output of this script is passed to Mesos.
 
-   We recommend the format for the script output be `fault_domain: region: name: <region>, zone: name: <zone>` We provide [fault domain detect scripts for AWS and Azure nodes](https://github.com/dcos/dcos/tree/master/gen/fault-domain-detect). For a cluster that has aws nodes and azure nodes you would combine the two into one script. You can use these as a model for creating a fault domain detect script for an on premises cluster.
+We recommend the format for the script output be:
 
-   <table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;"><b>Important:</b> This script will not work if you use proxies in your environment. If you use a proxy, modifications will be required.</td> </tr> </table>
+```json
```

Comment:
Should this be part of this PR?

Reply:
I have confirmed that these changes are already live. They have been having odd Jenkins caching issues between staging and master - not sure if this is related.

Reply:
https://docs.mesosphere.com/1.11/deploying-services/fault-domain-awareness/#installation-steps